americangut/util.py

```diff
@@ -214,7 +295,7 @@ def fetch_study(accession, metadata_path, fasta_path):
     md_f.write('\t'.join(header))
     md_f.write('\n')
     for sampleid, values in all_md.iteritems():
-        to_write = [values.get(k, "no_data") for k in header]
+        to_write = [values.get(k, "no_data").encode('utf-8') for k in header]
```

Comment:
likely due to some utf-8 chars slipping into EBI at some point during AG's lifecycle